tests/generator/generator.cpp

```diff
@@ -149,7 +149,7 @@ TEST(Generator, nonFirstOrderOdes)
 
     generator->processModel(model);
 
-    EXPECT_EQ_ISSUES_KINDS(expectedIsues, expectedCauses, generator);
+    EXPECT_EQ_ISSUES_CAUSES(expectedIsues, expectedCauses, generator);
```

Comment:
Spelling ...

Reply:
Actually, do a general search ... there are lots of them!  Should be `expectedIssues` (not ...Isues)

Reply:
Indeed, done as part of commit 0d779535.

Reply:
Copying/pasting... argh! :)